### PR TITLE
MovieClip.currentFrame renamed  because of confusing name

### PR DESCRIPTION
**Pull request by gprzybylowicz**
https://github.com/GoodBoyDigital/pixi.js/issues/1694

### CODE REVIEW

**Comment by RoryPicko** [comment]
surely currentFrame makes more sense as it is the current frame being rendered and not the totalTime that movieclip will take to play through. Your merge request doesn't make sense to me unless you rename totalTime to currentTime

**Comment by gprzybylowicz** [comment]
"doesn't make sense to me unless you rename totalTime to currentTime" - yep, maybe currentTime is better name, 
"only benefit is having an integer back instead of a float" - no, imagine that you have loop animation with 8 textures, after 1min currentTime is 60 but currentFrame is 4

**Comment by RoryPicko** [comment]
I don't know where my comment has gone!? but the variable makes more sense now and you raise a valid point about having frames/time as seperate entities!

**Comment by krzysztof-o** [comment]
I stumbled upon currentFrame yesterday and I confirm it is confusing. :+1:

File: src/extras/MovieClip.js [hunk 1]
```diff
@@ -56,13 +56,13 @@ function MovieClip(textures)
     this.onComplete = null;
 
     /**
-     * The MovieClips current frame index (this may not have to be a whole number)
+     * Elapsed time since animation has been started, used internally to display current texture
```
Comment (drkibitz) [inline src/extras/MovieClip.js]: If it is used internally, want to just make it internal? e.g. _currentTime, mark with @private

Comment (gprzybylowicz) [inline src/extras/MovieClip.js]: i was wondering about that, but you can still set/get this value that's why i used name without underscore.

Comment (drkibitz) [inline src/extras/MovieClip.js]: @gprzybylowicz this is a a convention in JavaScript in general. Marking with private and putting an underscore is means it is internal, and does not merrit a major version increase if the name changes, or any special treatment in deprecation etc..

Comment (gprzybylowicz) [inline src/extras/MovieClip.js]: @drkibitz . I know this convention you mentioned but I meant I wasn't sure if this var should be private or not:)   ok, i However  I've rethought and agree with you, it makes sense to make it privete,  will change it soon

File: src/extras/MovieClip.js [hunk 2]
```diff
@@ -56,13 +56,11 @@ function MovieClip(textures)
...
      *
-     * @member {number}
```
Comment (englercj) [inline src/extras/MovieClip.js]: Still want the `@member {number}` tag, even when private.

Comment (gprzybylowicz) [inline src/extras/MovieClip.js]: tag added

**Comment by drkibitz** [comment]
@gprzybylowicz thanks a lot, much less confusing IMO ;)

**Pull request merged**